src/lib/viz/basemap/create-map.ts

```diff
@@ -55,9 +55,12 @@ export function createMap(options: DeckGLMapOptions = DEFAULT_OPTIONS) {
 
   const deckMap = new (window.deck.DeckGL as any)({
     mapStyle: CartoMapStyle[chosenOptions.basemap.toUpperCase() as keyof typeof CartoMapStyle],
-    initialViewState: chosenOptions.view,
+    viewState: chosenOptions.view,
     container: chosenOptions.container,
-    controller: true
+    controller: true,
+    onViewStateChange: ({ viewState }) => {
+      deckMap.setProps({ viewState })
```

Comment:
Not sure about this. We'd be supplying the very same viewState to the map.

Reply:
I agree

Reply:
I think we would be forcing it to change state when "the state has changed". I mean, creating a never-ending state update loop, don't you think?

Reply:
As says in the comment from Victor below, the map does not update itself with pan and zoom events if `initialViewState` is not set. What it does is just send a `onViewStateChange` event with the new viewState so you can update it with `setProps`